src/schemas/session.schema.ts

```diff
@@ -0,0 +1,17 @@
+import { Session } from "@/models/session.model";
+import mongoose, { Schema } from "mongoose";
+import { ExerciseRepository } from "./exercise.schema";
+
+const sessionSchema = new Schema<Session>(
+  {
+    exercises: {
+      type: [ExerciseRepository],
+      required: true,
+    },
+  },
+  { timestamps: true }
+)
+
+
+export const SetRepository: mongoose.Model<Session> =
```

Comment:
Ahh I see the confusion. Since we're not storing sessions in their own collection, we don't need to create and export a "repository". 

But you will still want to export the `sessionSchema` so you can use it in the `userSchema` :)